github_pr.py

```diff
@@ -108,12 +109,64 @@ def github_list_prs(**args):
     elif 'head' in args and args['head']:
         prs = _load_prs_by_branch(**args)
         _print_prs(prs, **args)
+    elif 'filter' in args and args['filter']:
+        prs = github_filter_prs(**args)
+        #_print_prs(prs, **args)
     else:
         prs = repo.get_pulls()
         _print_prs(prs, **args)
     return list_return_obj
 
 
+def github_filter_prs(**args):
+    """
+    Filters prs to return only what is contained in the filters
+    """
+    gh = Github(args['token'])
+    repo = gh.get_repo(args['repo'])
+    all_prs = [{'pr':pr} for pr in repo.get_pulls()]
+    filters = {filter.split('=')[0]:filter.split('=')[1] for filter in args['filter'].split(',')}
+
+    if 'owner' in filters:
+        all_prs = [pr for pr in all_prs if filters['owner'] == pr['pr'].user.login]
+    if 'label' in filters:
+        pr_issue = []
+        for pr in all_prs:
+            issue = repo.get_issue(pr['pr'].number)
+            labels = [label.name for label in issue.get_labels()]
+            pr_issue.append({'pr':pr['pr'],'issue':issue,'labels':labels})
```

Comment:
yeah i haven't fully formatted yet. i was just looking for a cursery check to see if im headed in the right direction. looks like you commented below on that. Thanks!